pkg/scheduler/cache/cache.go

```diff
@@ -275,15 +275,38 @@ func (dvb *defaultVolumeBinder) AllocateVolumes(task *schedulingapi.TaskInfo, ho
 	return err
 }
 
+// AllocateVolumes allocates volume on the host to the task
```

Comment:
The function name is not match the reality.

Reply:
> The function name is not match the reality.

update